x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/TransactionFlyout/index.tsx

```diff
@@ -11,10 +11,13 @@ import {
   EuiTitle
 } from '@elastic/eui';
 import React from 'react';
+import { IUrlParams } from '..';
```

Comment:
fyi: my PR moved the urlParams interface: https://github.com/elastic/kibana/pull/23636/files#diff-07bc963256e81bcf166ef7e957b5aa11R150